### PR TITLE
Add Notepad2e

### PR DESCRIPTION
**Pull request by chirsz-ever**

### CODE REVIEW

**Review by HBelusca — CHANGES_REQUESTED** [review]
Can you show us an example that proves that this application works fine in ReactOS?

**Comment by chirsz-ever** [comment]
VirtualBox_ReactOS-0 4-2010022355

File: notepad2e.txt [hunk 1]
```diff
@@ -0,0 +1,11 @@
+[Section]
+Name = Notepad2e
+Version = 4.2.25
+License = 3-clause BSD
+Description = Notepad 2e is a fork of Notepad2 with word highlighting, simultaneous editing, math evaluation, un/grep, UAC elevation, complete regexps (PCRE), Lua lexers, DPI awareness and more 
+Size = 980 KiB
```
Comment (learn-more) [inline notepad2e.txt:6]: Size is not needed when SizeBytes is used.

Comment (chirsz-ever) [reply notepad2e.txt:6]: This should be mentioned in [the wiki](https://reactos.org/wiki/RAPPS).

**Comment by learn-more** [comment]
Thanks!

**Pull request merged**